install/runners/nunit-tools.wxs

```diff
@@ -0,0 +1,114 @@
+
+            <Feature Id="NUNIT.ENGINE"
+                     Level="1"
+                     Absent="disallow"
+                     Title="NUnit Engine"
+                     Description="Installs the NUnit engine">
+                <ComponentGroupRef Id="NUNIT.ENGINE" />
+            </Feature>
+
+            <Feature Id="NUNIT.CONSOLE"
+                     Level="1"
+                     Title="NUnit Console"
+                     Description="Installs the NUnit console runner">
+                <ComponentGroupRef Id="NUNIT.CONSOLE" />
+            </Feature>
+		</Feature>
+	</Product>
+
+    <!-- ***************************************************************** -->
+    <!-- **********       Define Directory Structure            ********** -->
+    <!-- ***************************************************************** -->
+    
+	<Fragment>
+		<Directory Id="TARGETDIR" Name="SourceDir">
+			<Directory Id="ProgramFilesFolder">
+                <Directory Id="INSTALLDIR" Name="NUnit.org">
+                    <Directory Id="TOOLS" Name="tools" />
+                </Directory>
+			</Directory>
+		</Directory>
+	</Fragment>
+
+	<Fragment>
+        <ComponentGroup Id="NUNIT.COMMON" Directory="TOOLS">
+            <Component Id="CHANGES.txt">
+                <File Id="CHANGES.txt" Source="..\..\CHANGES.txt" />
+            </Component>
+            <Component Id="LICENSE.txt">
+                <File Id="LICENSE.txt" Source="..\..\LICENSE.txt" />
+            </Component>
+            <Component Id="NOTICES.txt">
+                <File Id="NOTICES.txt" Source="..\..\NOTICES.txt" />
+            </Component>
+            <Component Id="nunit.ico">
+                <File Id="nunit.ico" Source="..\..\nunit.ico" />
+            </Component>
+        </ComponentGroup>
+        
+		<ComponentGroup Id="NUNIT.ENGINE" Directory="TOOLS">
+			<Component Id="NUNIT.ENGINE">
+                <File Id="nunit.engine.dll" 
+                      ProcessorArchitecture="msil" 
+                      Source="$(var.nunit.engine.TargetPath)" />
+			</Component>
+            <Component Id="NUNIT.ENGINE.API">
+                <File Id="nunit.engine.api.dll" 
+                      ProcessorArchitecture="msil" 
+                      Source="$(var.nunit.engine.api.TargetPath)" />
+                <File Id="nunit.engine.api.xml" 
+                      Source="$(var.nunit.engine.api.TargetDir)$(var.nunit.engine.api.TargetName).xml" />
+            </Component>
+		</ComponentGroup>
+
+        <ComponentGroup Id="NUNIT.CONSOLE" Directory="TOOLS">
+            <Component Id="NUNIT_AGENT">
+                <File Id="nunit_agent.exe" 
+                      ProcessorArchitecture="msil" 
+                      Source="$(var.nunit-agent.TargetPath)" />
+                <File Id="nunit_agent.exe.config" 
+                      Source="$(var.nunit-agent.TargetPath).config" />
+            </Component>
+            <Component Id="NUNIT_AGENT_X86">
```

Comment:
Agent isn't part of the console, it's part of the engine. When we have other runners, they will use the same engine components, including agent, as the console.